<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startpage</class>
 <widget class="QWidget" name="startpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>startforum</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 72, 166, 255), stop:0.885572 rgba(255, 185, 255, 255), stop:1 rgba(119, 119, 119, 255));</string>
  </property>
  <widget class="QPushButton" name="startbutton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(246, 245, 244);
border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>Start Tracker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logoutextbut">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>540</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 51, 59);</string>
   </property>
   <property name="text">
    <string>Logout and Exit</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="TextEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>841</width>
     <height>441</height>
    </rect>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
